packages/ilios-common/addon/components/course/rollover.js

```diff
@@ -71,6 +79,12 @@ export default class CourseRolloverComponent extends Component {
     this.selectedCohorts = this.selectedCohorts.filter((obj) => obj !== cohort);
   }
 
+  @action
+  updateStartDate(newStartDate) {
+    // if a date is forced that isn't allowed
```

Comment:
I'm not sure what this means?

Reply:
i copied that verbatim from the old rollover date picker component.

the only way i'm able to trigger an `undefined` value to be passed here is by interacting with the input field that the picker is tethered to. i.e. attempting to use backspace to clear out the field, or try copy/paste a new value in.  it doesn't change the input in the UI, but will fire the onChange event on the picker for some reason.